src/_locales/en_US/messages.json

```diff
@@ -91,6 +91,10 @@
         "message": "Prevent WebRTC from leaking local IP address",
         "description": "Checkbox label on the general settings page"
     },
+    "options_allow_sites_widget_replacement": {
```

Comment:
Would be nice to reorganize all the widget tab messages so that they are next to each in the logical order they appear on the page.